universe/vikram.hello/manifest.tt

```diff
@@ -0,0 +1,102 @@
+                                        preposition = ["with # rating"],
+                                        adjective_enum = {
+                                            G = ["suitable for children", "mild"],
+                                            R = ["violent", "bloody", "made for adults"]
+                                        }
+                                    }],
+                                    out rating_score : Number
+                                    #[min_number=0]
+                                    #[max_number=10]
+                                    #_[canonical = {
+                                        default = "base",
+                                        base = ["rating", "score"],
+                                        adjective_argmin = ["worst"],
+                                        adjective_argmax = ["best"]
+                                    }],
+                                    out actors : Array(Entity(vikram.hello:person))
+                                    #_[canonical = {
+                                        default = "preposition", 
+                                        base = ["actor", "actress"],
+                                        property = ["actor #", "actress #"],
+                                        preposition = ["by #", "with #"],
+                                        passive_verb = ["acted by #", "played by #", "acted with #"],
+                                        base_projection = ['actor', 'actress'],
+                                        passive_verb_projection = ['acted | by', 'played | by']
+                                    }],
+                                    out poster: Entity(tt:picture)
+                                    #[filterable = false]
+                                    #_[canonical="poster"])  
+    #_[canonical = ["movie"]]
+    #_[result={
+        top = "${id}",
+        list_concat = "${id}: ${release_date}.",
+    }];
+    monitorable list query person( out id: Entity(vikram.hello:person)
+                                    #_[canonical={
+                                        default = "base",
+                                        base = ["actor", "actress", "director", "producer"],
+                                        passive_verb = ["directed by #", "played by #"],
+                                    }],
+                                    out popularity : Number 
+                                    #_[min_number = 0]
+                                    #_[canonical = {
+                                        default = "base",
+                                        base = ["popularity", "fame"],
+                                        adjective_argmin=["least popular", "least famous"],
+                                        adjective_argmax=["most popular", "most famous"]
+                                    }])
+    #_[canonical = ["person"]]
```

Comment:
This should be "actor" rather than "person", because this query only returns actors